recipes/mykrobe/post-link.sh

```diff
@@ -1,43 +0,0 @@
-#!/bin/bash
-# This script downloads the panel data that mykrobe uses for its predictions
```

Comment:
why is this not needed anymore?

Reply:
Because the panel is no longer an external requirement of mykrobe. It is all handled internally now.